projects/igniteui-angular/src/lib/grids/filtering/grid-filtering.service.ts

```diff
@@ -146,7 +146,7 @@ export class IgxFilteringService implements OnDestroy {
         this.isFiltering = true;
 
         let expressionsTree;
-        if (expressions instanceof FilteringExpressionsTree) {
+        if ('operator' in expressions) {
```

Comment:
expressions can be null, so:
```suggestion
        if (expressions && 'operator' in expressions) {
```